File: NEIPG/PGDisclaimerViewController.m
//
//  PGDisclaimerViewController.m
//  NEIPG
//
//  Created by Yaogeng Cheng on 4/3/14.
//  Copyright (c) 2014 Yaogeng Cheng. All rights reserved.
//

#import "PGDisclaimerViewController.h"

@interface PGDisclaimerViewController ()

@end

@implementation PGDisclaimerViewController

- (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
{
    self = [super initWithNibName:nibNameOrNil bundle:nibBundleOrNil];
    if (self) {
        // Custom initialization
    }
    return self;
}
-(void)viewWillDisappear:(BOOL)animated
{
    //[_dWebView finalize];
    [super viewWillDisappear:animated];
}

- (void)viewDidLoad
{
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    [_dWebView loadRequest:[NSURLRequest requestWithURL:[NSURL fileURLWithPath:[[NSBundle mainBundle] pathForResource:@"disclaimer" ofType:@"htm"]isDirectory:NO]]];
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender
{
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

- (IBAction)disagreePressed:(id)sender {
    exit(0);
}
@end

File: Swift Conversion/Controller/PGDisclaimerViewController.swift
//
//  PGDisclaimerViewController.swift
//  NEIPG
//
//  Created by Gourav Sharma on 11/30/16.
//  Copyright © 2016 Mobileprogrammingllc. All rights reserved.
//

import UIKit
import Darwin

class PGDisclaimerViewController: UIViewController {

    
    @IBOutlet weak var dWebView: UIWebView?
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        dWebView?.loadRequest(URLRequest(url: URL(fileURLWithPath: Bundle.main.path(forResource: "disclaimer", ofType: "htm")!, isDirectory: false)))
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    @IBAction func disagreePressed(_ sender: Any) {
        exit(0)
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
